server/controllers/auth-controller.php

```
@@ -1,2 +1,53 @@
+require_once '../config/database.php';
+
+//Xử lý register
+if (isset($_POST['register'])){
+    $first_name = $_POST['first_name'];
+    $last_name = $_POST['last_name'];
+    $email = $_POST['email'];
+    $uuid = uniqid();
+    $password = password_hash($_POST['password'], PASSWORD_DEFAULT);
+
+    //Kiểm tra xem email có tồn tại hay chưa
+    $checkEmail = $conn->query("SELECT email FROM users WHERE email = '$email'");
```

PHPStan (2.1.46)

[error] Variable $conn might not be defined.


(variable.undefined)